File: eRecipes/UI/erecipes_mobile/lib/models/omiljeni_recept.dart
import 'package:erecipes_mobile/models/recept.dart';
import 'package:json_annotation/json_annotation.dart';

part 'omiljeni_recept.g.dart';

@JsonSerializable()
class OmiljeniRecept{
  int? omiljeReceptId;
  int? korisnikId;
  int? receptId;
  DateTime? datumDodavanja;
  Recept? recept;
  
   OmiljeniRecept();
  
 factory OmiljeniRecept.FromJson(Map<String,dynamic> json)=> _$OmiljeniReceptFromJson(json);

  Map<String,dynamic> toJson() => _$OmiljeniReceptToJson(this);
}

File: eRecipes/eRecipes.Model/OmiljeniRecept.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace eRecipes.Model
{
    public class OmiljeniRecept
    {
        public int OmiljeniReceptId { get; set; }
        public int? KorisnikId { get; set; }
        public int? ReceptId { get; set; }
        public Recept Recept { get; set; }
        public DateTime? DatumDodavanja { get; set; }
    }
}
